sapper-dev-client.js

```diff
@@ -13,7 +13,7 @@ function check() {
 export function connect(port) {
 	if (source || !window.EventSource) return;
 
-	source = new EventSource(`http://${window.location.hostname}:${port}/__sapper__`);
```

Comment:
The [`window.location.protocol`](https://webplatform.github.io/docs/apis/location/protocol/) is of the format `http:` or `https:` so i think you should remove the extra colon.